impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -190,6 +190,25 @@ std::shared_ptr<Texture> MakeBlurSubpass(
   return out_texture;
 }
 
+// This function was derived with polynomial regression when comparing the
+// results with Skia.  Changing the curve below should invalidate this.
+//
+// The following data points were used:
+//   0 | 1
+//  75 | 0.8
+// 150 | 0.5
+// 300 | 0.22
+// 400 | 0.2
+// 500 | 0.15
+//
+// TODO(gaaclarke): Maybe we should just change this to a linear function.
+Scalar ScaleSigma(Scalar sigma) {
```

Comment:
I chose to duplicate this instead trying to share with the old blur.  I want to keep it the same for now so that we get better apples to apples comparison with the benchmarks when we move it over.  I think there is a weirdness where the minima of this function isn't at 500 so the blur can slightly dip and come back.  Also the value at zero should be 1.0, not 1.02.